lib/matplotlib/backends/backend_qt5.py

```diff
@@ -613,6 +613,14 @@ def _init_toolbar(self):
             self.layout().setSpacing(12)
             self.setMinimumHeight(48)
```

Comment:
Hmm, good point. It does appear to do something eventually because the toolbar _is_ 48px high when drawn; it must have to go through some resize events or something before it's updated.